functions/functionsv2/helloauditlog/hello_auditlog.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_log_cloudevent]
+
+// Package helloworld provides a set of Cloud Functions samples.
+package helloworld
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/cloudevents/sdk-go/v2/event"
+	auditevents "github.com/googleapis/google-cloudevents-go/cloud/audit/v1"
```

Comment:
blocking: Unfortunately this library is not GA and we shouldn't use it yet. I'm curious how you found your way to this library, and what we might do to prevent others for deciding to use it?

Reply:
I'll remove the use of this library, and let you know when i'm ready for another review.

There is a documentation page that appears to conflate the two things called "cloud event": https://cloud.google.com/eventarc/docs/cloudevents#open-source
there are additional references in the Common Events section that link to protobufs, and the google-cloudevents repo docs.